hmda/src/main/scala/hmda/query/ts/TsComponent.scala

```diff
@@ -0,0 +1,96 @@
+package hmda.query.ts
```

Comment:
This entire class is quite similar to `TransmittalSheetComponent` in hmda-analytics as well as `TransmittalSheetTable` in hmda-regulator. Maybe not as part of this PR but we should extract all these into common rather than duplicating the code base. Maybe a separate ticket.

Reply:
https://github.com/cfpb/hmda-platform/issues/2579